packages/ai/src/ai/modules/remote/remote.py

```
@@ -1,4 +1,4 @@
-from typing import Any, Dict, List
+from typing import Any, Dict, List, Optional
```

Ruff (0.15.7)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)